product-specifications/example-2.md

```diff
@@ -0,0 +1,21 @@
+# Interactive Animal Identification Tool (IAIT)
+
+This tool will help users identify animals by asking a series of questions and then provide a list of most likely animals based on the user's responses. This tool will be used by wildlife conservationists to identify animals they encounter.
+
+## User Flow
+
+1. Users answer a series of questions about the observed animal:
+    - What color is the animal?
```

Comment:
maybe create colour boxes with a range of colours to help filter more as probably an animal may have mixed colours due to  dust

Reply:
I like color box input here!